src/tools/html-checker/main.rs

```diff
@@ -0,0 +1,65 @@
+use std::env;
+use std::path::Path;
+use std::process::{Command, Output};
+
+fn check_html_file(file: &Path) -> bool {
+    let mut to_mute = vec!["PROPRIETARY_ATTRIBUTE"];
```

Comment:
Can you add a comment here of which tags this is ignoring? I see these locally:
```
line 1 column 599 - Warning: <link> proprietary attribute "disabled"
line 4 column 628 - Warning: <select> proprietary attribute "autocomplete"
```

Reply:
Sure!